Source/WebCore/page/FrameView.h

```diff
@@ -27,21 +27,13 @@
 #include "AbstractFrameView.h"
 #include "AdjustViewSizeOrNot.h"
 #include "Color.h"
-#include "ContainerNode.h"
 #include "FrameViewLayoutContext.h"
-#include "GraphicsContext.h"
```

Comment:
Some Windows code seemed to expect this, maybe just `WebProcess\WebPage\CoordinatedGraphics\LayerTreeHostTextureMapper.cpp`.